tests/sharding/mainchain_handler/fixtures.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+import rlp
+
+from web3 import (
+    Web3,
+    # HTTPProvider,
+)
+
+from web3.providers.eth_tester import EthereumTesterProvider
+
+from eth_tester import EthereumTester
+
+from eth_tester.backends.pyevm import PyEVMBackend
+
+# from evm.chains.sharding.mainchain_handler.config import (
+#     DEFAULT_RPC_SERVER_URL,
+# )
+
+from evm.chains.sharding.mainchain_handler.mainchain_handler import (
+    MainchainHandler,
+)
+
+
+class TesterMainchainHandler(MainchainHandler):
+    def mine(self, number):
+        self.w3.testing.mine(number)
+
+    def direct_tx(self, tx):
+        raw_tx = rlp.encode(tx)
+        raw_tx_hex = self.w3.toHex(raw_tx)
+        try:
+            tx_hash = self.w3.eth.sendRawTransaction(raw_tx_hex)
+        except ValueError:
+            # FIXME: if `sendRawTransaction` is not implemented, `ValueError` is raised
```

Comment:
You should be good to go using `sendRawTransaction` directly with the latest `eth-tester` release.

Reply:
May I remove this after https://github.com/ethereum/web3.py/pull/520 is merged? Otherwise we might probably get errors.

Reply:
Yes, lets see if we can't get @carver to merge and release that today.